model/metrics/gen-ai.yaml

```diff
@@ -43,3 +43,19 @@ groups:
           The `error.type` SHOULD match the error code returned by the Generative AI provider or the client library,
           the canonical name of exception that occurred, or another low-cardinality error identifier.
           Instrumentations SHOULD document the list of errors they report.
+  - id: metric.gen_ai.server.latency.time_per_output_token
+    type: metric
+    metric_name: gen_ai.server.latency.time_per_output_token
+    brief: 'Mean time per output token generated'
```

Comment:
why `mean time` and not just time? I am not quite sure how the `mean` will be calculated here

Reply:
Agree, mean is a little confusing. Fixed it. Mainly it is the (total tokens generated per request) / (request latency - time to first token). This is then separated into buckets.